index.html

```diff
@@ -0,0 +1,83 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css\style.css"/>
+<body>
+ <div id="header"><h1 id="title">Microverse Survey Form</h1>
+  <p id="description">It's all because of you that we got this far.Thank you!</p>
+ </div>
+  <form id="survey-form">
+   <label id="name-label">Name</label>
+   <br>
+   <input id="name" type="text" placeholder="Enter your name" required>
+   <br>
+   <label id="email-label" >Email</label>
+   <br>
+   <input id="email" type="email" placeholder="Enter your email" required>
+   <br>
+   <label id="number-label">Age</label>
+   <br>
+     <input id="number" type="number" placeholder="Enter your age" min=15 max=125 >
+     <br>
+     <label>What's your occupation</label>
+     <br>
+     <select id="dropdown">
+       <option> Choose from below</option> 
+       <option>Software Engineer</option>
+       <option>Teacher</option>
+       <option>Banker</option>
+       <option>Student</option>
+     </select>
+     <br>
+     <label>Are you willing to work with Microverse</label>
+     <br>
+     <input value="Absolutely" name="decision" type="radio"> 
+     <label>Absolutele</label>
+     <br>
+     <input value="Mayb" name="decision" type="radio">
+     <label>Maybe</label>
+     <br>
+     <input value="Not sure" name="decision" type="radio">
+     <label>Not sure</label>
+     <br>
+     <legend>what pragramming languages do you prefer(choose all that apply)</legend>
```

Comment:
I kindly suggest that you a sentence or a question with a capital letter. For instance, in line 45, it should be "What programming language do you prefer?" not "what programming language do you prefer". Good grammar makes your work professional and hence attracts more clients.